pkgs/by-name/ol/olympus/update.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq common-updater-scripts nixfmt-rfc-style
+
+set -eu -o pipefail
+
+attr=olympus
```

Comment:
```suggestion
attr=olympus.passthru.unwrapped
```
Make this change after you exposed the unwrapped package in `passthru`.